api/templates/application.xml

```diff
@@ -34,6 +34,7 @@
             <ValidationResponses Version="1" Type="FormInstanceSpecific" />
           </FormInstanceSpecific>
           <Pooled>
+            {{tmpl "additional-comments.xml" .Package.Comments}}
```

Comment:
let's add this at the end of the list instead of the beginning so this list maintains its order relative to the actual sections